### DISCUSSION TITLE
react-native source map of hermes bundle with firebase crashlytics

### DISCUSSION THREAD

**Discussion by MaxToyberman** [invertase/react-native-firebase#5390] [Q&A] [answered]
Hi,
I've been using Crashlytics and source map cli for a long time to find out where is my error located.
recently I have started to use hermes engine and now all errors are on line 1 see example:

```
Non-fatal Exception: io.invertase.firebase.crashlytics.JavaScriptError: Cannot read property 'slice' of undefined
       at .anonymous(address at index.android.bundle:1:1540468)
       at .anonymous(address at index.android.bundle:1:1543708)
       at .anonymous(address at index.android.bundle:1:1563240)
       at .anonymous(address at index.android.bundle:1:1563134)
```

any ideas ? 

Thank you very much

**Comment by mikehardy** [comment]
Hi there! Resolution of crash traces from the crashlytics console down to actual lines of code in dynamic languages is difficult.
Nothing I offer here is a concrete implementation but you ask for ideas so perhaps this will help.
You must tag your source exactly as built, then maintain the ability to generate source maps (or save the source maps) for each transpilation / compliation step

You may wish to follow here, it is a better place for the question as this is really a Hermes question, not a react-native-firebase one: https://stackoverflow.com/questions/67835466/react-native-source-map-of-hermes-bundle-with-firebase-crashlytics

And this may be interesting: https://docs.bugsnag.com/build-integrations/js/source-maps-react-native/#hermes - note the location of the bundle map

**Reply by MaxToyberman** [reply]
@mikehardy thank you for your reply.
Yes the issue is Hermes, previously i could see the line number and column and now i don't.

is there any guide/tutorial to do it with Hermes.

**Reply by mikehardy** [reply]
I have provided all the information I have

**Reply by mikehardy** [reply]
You still have the line number and column by the way - that's clear. It's just one really big line.

**Reply by MaxToyberman** [reply]
@mikehardy  thanks

**Comment by MaxToyberman** [comment, marked as answer]
i found the solution, 

please follow this instructions:

https://reactnative.dev/docs/next/symbolication

**Reply by Rotarex-Guillaume-Michael** [reply]
dead link :(
